cluster/cluster_members.go

```diff
@@ -159,14 +159,14 @@ END;
 
 // UpdateClusterMemberSchemaVersion sets the schema version for the cluster member with the given address.
 // This helper is non-generated to work before generated statements are loaded, as we update the schema.
-func UpdateClusterMemberSchemaVersion(ctx context.Context, tx *sql.Tx, internalVersion uint64, externalVersion uint64, address string) error {
+func UpdateClusterMemberSchemaVersion(ctx context.Context, tx *sql.Tx, internalVersion uint64, externalVersion uint64, memberName string) error {
```

Comment:
I didn't like this solution partially for this reason; if any downstream users are using this function they won't be alerted to the change of semantics since the type doesn't change. That said neither Ceph nor OVN use this function.

Reply:
These functions shouldn't be used externally anyway. After this one gets merged I'm planning to send up 1 more to lock down the database a bit with an interface.